pom.xml

```diff
@@ -751,7 +751,7 @@ and
     <profile>
       <id>lts</id>
       <properties>
-        <jenkins.version>2.452.3</jenkins.version>
+        <jenkins.version>2.462.1</jenkins.version>
```

Comment:
Done in `jenkinsci/bom` in https://github.com/jenkinsci/bom/pull/3447/commits/b6f3277b1455c4913a1e0c3f59f7463c30358928.